### DISCUSSION TITLE
Suggest an Edit

### DISCUSSION THREAD

**Discussion by rocodcli** [GitbookIO/community#395] [Feature Requests]
Hello, 

the PR process would benefit from a "suggest edit" functionality.
Today you can comment on a block but if the block is of significant length you have to find the needle in a haystack to update the text.
Also, it is possible to modify the text of someone else's PR which is very confusing to me for a "Review Process"

**Comment by thebiglabasky** [comment]
Thanks for your input! That's an interesting suggestion.

For now the way to suggest edits requires indeed to create a new change request so the author can enable the diff view and see the proposed changes.
That's more involved than what Google Docs propose, to ensure that any change to a public space can be thoroughly reviewed.
